CONTRIBUTING.md

```diff
@@ -0,0 +1,6 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change.
+
+see https://stfc.atlassian.net/wiki/spaces/SC/pages/232554517/DevOps+-+Git+Conventions for details
```

Comment:
I think maybe remove the link here

Reply:
@Koldstream Is getting this moved to our user facing docs, when that's done if he could post the new link as a comment?

Reply:
Once Jacob has approved, I am getting John to move the pages to CloudKB. Once that is done I'll put the new url in here :)

Reply:
@meoflynn should we merge this one then - since @Koldstream can make a new PR to change the link

Reply:
Ready!

```suggestion
see https://stfc.atlassian.net/wiki/spaces/CLOUDKB/pages/283770979/Coding+Conventions for details
```